asset/asset.go

```diff
@@ -36,6 +45,15 @@ func (s SerializedKey) CopyBytes() []byte {
 	return c
 }
 
+// CopyBytes returns a copy of the underlying array as a byte slice.
+func (s SchnorrKey) CopyBytes() []byte {
+	return fn.CopyBytes(s[:])
+}
+
+func (s SchnorrKey) IsZero() bool {
+	return bytes.Equal(s[:], ZeroGroupKey[:])
```

Comment:
Since they're both of the same type and an array, can just use `s == ZeroGroupKey` (which I imagine is faster as well).